examples/js/loaders/GLTFLoader.js

```diff
@@ -172,6 +180,14 @@ THREE.GLTFLoader = ( function () {
 
 			} );
 
+			if ( this.lazy ) {
+
+				onLoad( { parser: parser } );
+
+				return;
+
+			}
```

Comment:
Would you want to also return the `json`  object here? I imagine users will want to use that to access `extensions` data, or to list the available materials.

Reply:
The `json` object is available via `parser.json`. I think this lines up well with the non-lazy method.

When specifying `lazy`:
```
onLoad( { parser: parser } )
```

When using the default non-lazy method:
```
onLoad( {
  scene: scene,
  scenes: scenes,
  cameras: cameras,
  animations: animations,
  asset: json.asset,
  parser: parser,
  userData: {}
} )
```